src/app/lib/config/services/portuguese.js

```diff
@@ -205,12 +205,6 @@ export const service = {
           href: 'https://www.bbc.com/portuguese/institutional-36202456',
           text: 'Fale com a BBC',
         },
-        {
-          href:
-            'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
-          text: 'AdChoices / Do Not Sell My Info',
```

Comment:
We need to have this additional footer link for all services. Unfortunately the reference spreadsheet hasn't been updated to reflect this. We should ask Jon to update that.

Reply:
Thanks 👍 Added it back